### PR TITLE
Extend and update documentation

### PR DESCRIPTION
**Pull request by ghost**
This PR extends and update the documentation to provide more background for users - in particular w.r.t. terminology validation - and to reflect the changes introduced by recent PRs.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -320,6 +298,54 @@ without the actual codes. Please only reference the SNOMED CT codes after instal
...
   ```
 
+## Using external terminology servers
+
+You can configure one or more external terminology servers using the configuration for the validator-server module (see below for details). This is done by providing the base URLs on which the servers can be reached.
+
+# Web interface
+
+The `compass-conformance-server` comes with a one-page web interface to validate resources. The issues of the [operation
+outcome](https://www.hl7.org/fhir/operationoutcome.html) will be displayed will be displayed according their severities.
```
Comment (TimWalz) [inline docs/README.md]: Duplicate "will be displayed" in text

**Review by TimWalz — CHANGES_REQUESTED** [review]
Found only a minor duplication in the text, the rest looks good!

File: docs/README.md (hunk 1 above)
Comment (ghost) [reply docs/README.md]: 👍 , will fix

**Comment by ghost** [comment]
@TimWalz Ready for re-review 😄

**Review by TimWalz — APPROVED** [review]

**Pull request merged**